src/app/content/highlights/selectors.ts

```diff
@@ -35,3 +49,30 @@ export const summaryIsLoading = createSelector(
   localState,
   (state) => state.summary.loading
 );
+
+const loadedCountsPerPageInSummary = createSelector(
+  summaryHighlights,
+  flow(
+    values,
+    map(toPairs),
+    flatten,
+    fromPairs,
+    mapValues(size)
+  )
+);
+
+const filteredTotalCountsPerPageInSummary = createSelector(
+  localState,
+  (state) => state.summary.filteredTotalCounts
+);
+
+export const totalCountsPerPageInSummary = createSelector(
```

Comment:
this selector isn't currently used, but should be used to disable chapter filters that have no highlights in any of their pages